### DISCUSSION TITLE
Using an older version of extension (ACM Journal in my case)

### DISCUSSION THREAD

**Discussion by Sumidu** [quarto-dev/quarto-cli#2811] [Q&A]
After updating to the current ACM Journal extension, I get an error (reported it on the repository already).
Nevertheless, I am wondering whether I **can use an older version of an extension. And if so, how?** Is there something like a previous commit ID?

**Comment by Sumidu** [comment]
I just want to add this here, too. 
The problem was with Quarto v1.1.xx ACM Journal needs v.1.2.xx
Still, can one use an older version of a template?

**Reply by cscheid** [reply]
It is possible to refer to a tag in the `quarto install` command, but `acm` doesn't have versions yet, since this is still under active development.

If you're adamant about doing so, the easiest way is to clone the repository, checkout the commit ID you want, and then create the directory structures yourself. `quarto use template` does very little besides copying the content in the repository.
